packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -80,6 +80,7 @@ export default defineComponent<KottiButton.PropsInternal>({
 	display: inline-flex;
 	align-items: center;
 	justify-content: center;
+	contain: content;
```

Comment:
1. How did you find out about this? I've never seen this feature
2. This doesn't appear to be supported by Safari. Do you have an alternative solution? If no, then this is better than nothing

Reply:
1. Nothing too sophisticated, I just did highly uncoordinated experiments in the browser dev tools
2. Changed the fix to use `vertical-align`, that should have a bit of a broader support

Reply:
I am not entirely sure why `vertical-align: middle` works (along with a few alternative values like `text-bottom` or `text-top`). Playing around with it I noticed that if you unset `display: inline-flex` for `.kt-button` the texts of the buttons would align. So I guess it's something to do with the order in which `vertical-align` and `flex` is calculated and drawn by browsers.